backend/baseclass.pm

```diff
@@ -343,6 +343,11 @@ sub check_select_rate ($buckets, $wait_time_limit, $hits_limit, $id, $time) {
 sub _invoke_video_encoder ($self, $pipe_name, $display_name, @cmd) {
     my $pid = open($self->{$pipe_name}, '|-', @cmd);
     my $pipe = $self->{$pipe_name};
+    if ($bmwqemu::vars{XRES} && $bmwqemu::vars{YRES}) {
```

Comment:
Could you put a part of your explanation from the commit message in here as a comment to explain the reasoning for the code?

Reply:
Sure. I also wonder if it's a good idea to try increasing the pipe size even for default 1024x768? Right now I intentionally limited it to non-default resolutions only (to not emit the warning in most cases), but maybe it makes sense in any case? I haven't tried if improvement on 1024x768 would be noticeable at all, but in theory it may help too.

Reply:
I would be very much interested if you can do any benchmarking showing any influence?

Reply:
It looks like I don't have system slow enough for this to be a bottleneck with 1024x768 resolution. On my RPi setup both with and without this change the `Passing remaining frames to the video encoder` stage takes about 3s.